examples/websocket_client.py

```
@@ -0,0 +1,281 @@
+            }
+            
+            try:
+                await self.websocket.send(json.dumps(leave_message))
+                await self.websocket.close()
+            except:
+                pass  # Ignore errors during cleanup
+        
+        logger.info("📴 Disconnected from WandererKills WebSocket")
+
+
+async def example():
```

Ruff (0.11.9)

Missing return type annotation for public function `example`

Add return type annotation: `None`

(ANN201)